### ISSUE TITLE
IDE won't start

### ISSUE THREAD

**Issue by asdf2154s** [mono/monodevelop#6530]
IDE won't start on ubuntu 18.04.
There is no any error message. Just mouse spinner for a few seconds, and then nothing.
Is there any log file to check?

**Comment by Therzok** [comment]
Yup. https://www.monodevelop.com/developers/articles/user-profiles/#logs

**Comment by rdrpenguin04** [comment]
I got this too. Here's my log: [https://pastebin.com/4WDzZtJp](https://pastebin.com/4WDzZtJp)
Don't mind the pwndbg, that's a gdb plugin I use.
